lib/utils.py

```diff
@@ -329,3 +331,30 @@ def is_balena_app():
     :return: bool
     """
     return bool(getenv('RESIN', False)) or bool(getenv('BALENA', False))
+
+
+def is_wott_integrated():
```

Comment:
@eugenosm this won't work well across runtime environments (such as Docker). It's also very hackish. Just do if `WOTT_PATH` exists, then assume the agent is installed.

Reply:
ok